server/modules/collectImport/service/collectImport/metaImportJobs/taxonomiesImportJob.js

```diff
@@ -149,8 +148,8 @@ class TaxonomiesImportJob extends Job {
       this.addError({
         [keysError.code]: {
           valid: false,
-          errors: [`duplicate code ${code}; row: ${this.currentRow} duplicate row: ${rowDuplicateCode}`]
-        }
+          errors: [{ key: 'duplicateCode', params: { code, row: this.currentRow, duplicateRow: rowDuplicateCode } }],
```

Comment:
it would be preferable to have duplicateCode to keysError constant as the other key errors

Reply:
Yes right